rest_framework/schemas/generators.py

```diff
@@ -452,3 +475,55 @@ def get_keys(self, subpath, method, view):
 
         # Default action, eg "/users/", "/users/{pk}/"
         return named_path_components + [action]
+
+
+class OpenAPISchemaGenerator(BaseSchemaGenerator):
+
+    def get_info(self):
+        info = {
+            'title': self.title,
```

Comment:
Nice work there, grats.
It looks like this breaks the `OpenAPIRenderer` at https://github.com/encode/django-rest-framework/blob/master/rest_framework/renderers.py#L1017-L1018 where the structure is expected to be an object and not a dict. Beside, the renderer assumes `description` to be mandatory.

Reply:
OK, so the renderer there will be deprecated here. (It takes a CoreAPI document and maps it to OpenAPI)

I’ll begin on the new docs this week. And begin moving out the CoreAPI code into a separate module, so it’s all clearer. 

Right now, just dump the Schema straight to JSON or YAML if you want to serialise it. (SchemaView is not currently functional, but is on the list. Possible entry point for assistance there... 🙂)